.github/workflows/tests.yml

```diff
@@ -4,12 +4,14 @@ name: Test suite
 
 on:
   pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
     branches: [ master ]
 
 jobs:
   lint-unittests-docs:
     name: Lint, Unit tests and Docs
     runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.draft }}
```

Comment:
I tried to use `if: ${{ !github.event.pull_request.draft || matrix.python-version == 3.9 }}` (#21) to run these tests for python 3.9 only. Unfortunately, it doesn't work :-1:.